addons/modules/fnc_getStringPos.sqf

```diff
@@ -0,0 +1,47 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getStringPos
+
+Description:
+    A function used to get the position of a string with a passed typeName.
+
+Parameters:
+    - Number (Number from 0 to 5)
+        0 - Return 0 to decide what to do later in script
+        1 - Object or location
+        2 - Group
+        3 - Array
+        4 - Marker
+        5 - Task
+    - String (XYZ, Object, Location, task, group, or Marker)
+
+Example:
+    (begin example)
+    [1, "player"] call CBA_fnc_getStringPos;
+    (end)
+
+Returns:
+    Position [0,0,0] or 0
+
+Author:
+    WiredTiger
+
+---------------------------------------------------------------------------- */
+
+//SCRIPT(getStringPos);
+
+params [
+    ["_type",0],
+    ["_string",""],
+    ["_stringPos",0]
+];
+
+switch (_type) do {
```

Comment:
@jonpas As it works right now it should only be one parameter passed so that function is not needed for this particular instance, but I could use that for my original idea of using the module to spawn groups instead of syncing them.